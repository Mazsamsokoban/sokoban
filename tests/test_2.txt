loadtest 2
showstate w
move w right
showmaze
showstate w