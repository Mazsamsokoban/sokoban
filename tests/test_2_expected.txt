f1:w hole:null 

name:w
field:f1
direction:Neutral
points:0
pushforce:10.0

f1:null hole:null 

name:w
field:null
direction:Right
points:0
pushforce:10.0

